src/nix/daemon.cc

```diff
@@ -538,6 +550,15 @@ struct CmdDaemon : StoreCommand
             }},
             .experimentalFeature = Xp::DaemonTrustOverride,
         });
+
+        addFlag({
+            .longName = "process-ops",
+            .description = "Forces the daemon to process received commands itself rather than forwarding the commands straight to the remote store. This is useful ",
```

Comment:
```suggestion
            .description = "Forces the daemon to process received commands itself rather than forwarding the commands straight to the remote store.",
```

Reply:
True, but then I thought that this flag is quite more general and referring to the mounted ssh store here would make it a bit more confusing.